tests/integration/profiling/rule_based_profiler/test_run_profiler_notebook.py

```diff
@@ -246,3 +246,45 @@ def test_run_multibatch_configured_asset_example(tmp_path):
             "great_expectations/uncommitted/validations/example_configured_suite",
         ]
         clean_up_test_files(base_dir=base_dir, paths=paths_to_clean_up)
+
+
+def test_run_multibatch_sql_asset_example(tmp_path):
+    """
+    What does this test and why?
+    One of the resources we have for multi-batch Expectations is a Jupyter notebook that explains/shows the components
+    in code. This test ensures the codepaths and examples described in the Notebook actually run and pass, nbconvert's
+    `preprocess` function.
+    """
+    base_dir: str = file_relative_path(
+        __file__, "../../../test_fixtures/rule_based_profiler/example_notebooks"
+    )
+    notebook_path: str = os.path.join(base_dir, "MultiBatchExample_SqlExample.ipynb")
+    # temporary output notebook for traceback and debugging
+    output_notebook_path: str = os.path.join(
+        tmp_path, "MultiBatchExample_SqlExample_executed.ipynb"
+    )
+    with open(notebook_path) as f:
+        nb: NotebookNode = nbformat.read(f, as_version=4)
+
+    ep: ExecutePreprocessor = ExecutePreprocessor(timeout=60, kernel_name="python3")
```

Comment:
This is probably fine but 60s seems like a long time.

Reply:
I think this was pulled directly from the `nbformat` documentation.. On closer inspection I agree Maybe it can be dropped by 1/2